test/powershell/Modules/Microsoft.PowerShell.Management/Get-ChildItem.Tests.ps1

```diff
@@ -142,6 +145,14 @@ Describe "Get-ChildItem" -Tags "CI" {
             (Get-ChildItem -LiteralPath $TestDrive -Depth 1 -Exclude $item_a).Count | Should Be 5
         }
 
+        It "Should list files in directory contains special char" {
+            $null = New-Item -Path $TestDrive -Name $specialDirName -ItemType Directory -Force
+            $specialPath = Join-Path $TestDrive $specialDir
+            $null = New-Item -Path $specialPath -Name file1.txt -ItemType File -Force
+            $null = New-Item -Path $specialPath -Name file2.txt -ItemType File -Force
```

Comment:
Can we maybe remove the `-Force` here? Adding `-Force` would make the `New-Item -Path $TestDrive -Name $specialDirName -ItemType Directory -Force` line above not relevant, as `-Force` will force creating the folders.

Reply:
Do you mean remove `-Force` from all 3 lines? or just the last 2 that try to create file only?

Reply:
I mean removing `-Force` for line 151 and 152, so it will test if `$specialPath` can correctly resolve to the folder.